src/components/Navbar.jsx

```diff
@@ -69,12 +69,19 @@ const Navbar = () => {
                 <Link to={"/about-us"}>About Us</Link>
                 <div className={"" + (location.pathname === '/about-us' ? "active-link" : "")}></div>
               </li>
-
-              <li className="nav-item home">
-                <Link to={"/contact"}>Contact</Link>
-                <div className={"" + (location.pathname === '/contact' ? "active-link" : "")}></div>
-              </li>
-
+              {/* Render Contact Us in Navabr only for Login and Register Pages */}
+              {(location.pathname === "/user/login" ||
+                location.pathname === "/user/register") && (
+                <li className="nav-item home">
+                  <Link to={"/contact"}>Contact</Link>
+                  <div
+                    className={
+                      "" +
+                      (location.pathname === "/contact" ? "active-link" : "")
+                    }
+                  ></div>
+                </li>
+              )}
 
```

Comment:
You need to make it visible for `/clubs/register` and `/clubs/login` too

Reply:
Will do the changes soon!